test/metabase/util/malli_test.clj

```diff
@@ -32,31 +34,30 @@
     (testing "outer schema"
       (let [special-lt-4-schema (mu/with-api-error-message
                                   [:fn less-than-four-fxn]
-                                  "Special Number that has to be less than four")]
-        (is (= [:fn {:description "Special Number that has to be less than four",
-                     :error/message "Special Number that has to be less than four"}
-                less-than-four-fxn]
-               (mc/form special-lt-4-schema)))
+                                  (deferred-tru "Special Number that has to be less than four error")
+                                  (deferred-tru "Special Number that has to be less than four description"))]
 
-        (is (= ["Special Number that has to be less than four"]
+        (is (= [(deferred-tru "Special Number that has to be less than four description")]
                (me/humanize (mc/explain special-lt-4-schema 8))))
 
-        (is (= "Special Number that has to be less than four"
-               (umd/describe special-lt-4-schema)))))
+        (testing "describe should be user-localized"
+          (is (= "Special Number that has to be less than four error"
+                 (umd/describe special-lt-4-schema)))
```

Comment:
these look backward: the call to humanize should return "... error" and the call to umd/describe should return "... description" right?

Reply:
to reiterate: I think just the arg order here is backward. But I think it's due to the arg names being a bit unclear. See the comment in metabase.malli.util.

Reply:
haha yea it was backward, I fixed it in: https://github.com/metabase/metabase/pull/28053/commits/2d989c4102f32d3c6b4303f8af5d2bbe01cae4e0

my left and right brain were switched for a moment there...